core/src/main/java/com/yahoo/ycsb/generator/CounterGenerator.java

```diff
@@ -32,23 +32,18 @@
 	public CounterGenerator(int countstart)
 	{
 		counter=new AtomicInteger(countstart);
-		setLastInt(counter.get()-1);
```

Comment:
It doesn't make sense to call `setLastValue` here. This implementation implicitly ties (and did before my changes too) the last value to current one, as opposed to depending on the side effect of calling `setLastValue` (setting a field in base class).